src/components/FeatureViews/Dashboard/DashboardNotes.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+// import {useSelector} from "react-redux";
+
+import * as hooks from "actions/hooks";
+import Loading from "components/Loading";
+import Notes from "./../Notes/Notes";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/styles";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        [theme.breakpoints.down('md')]: {
+            height: "68vh",
+            overflowY: "scroll",
+            marginTop: "72px",
+        }
+    }
+}))
+
+const DashboardNotes = (owner) => {
+    const classes = useStyles();
+
+    const ownerID = owner.id;
+    const ownerType = 'account'
+    const adminStatus = hooks.useAdmin();
+
+if (hooks.isLoading(adminStatus)) {
+    return (
+        <Loading
+            loadingText="NOTES ARE LOADING"
+            small />
+    );  
+}
```

Comment:
Fetching the admin is not necessary to fetch the notes, with graphQL. Remove this.